src/Sentry/PlatformAbstractions/RuntimeInfo.cs

```diff
@@ -102,13 +102,20 @@ internal static void SetNetCoreVersion(Runtime runtime)
 
         internal static Runtime? GetFromRuntimeInformation()
         {
-            // Preferred API: netstandard2.0
-            // https://github.com/dotnet/corefx/blob/master/src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs
-            // https://github.com/mono/mono/blob/90b49aa3aebb594e0409341f9dca63b74f9df52e/mcs/class/corlib/System.Runtime.InteropServices.RuntimeInformation/RuntimeInformation.cs
-            // e.g: .NET Framework 4.7.2633.0, .NET Native, WebAssembly
-            var frameworkDescription = RuntimeInformation.FrameworkDescription;
+            try
+            {
+                // Preferred API: netstandard2.0
+                // https://github.com/dotnet/corefx/blob/master/src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs
+                // https://github.com/mono/mono/blob/90b49aa3aebb594e0409341f9dca63b74f9df52e/mcs/class/corlib/System.Runtime.InteropServices.RuntimeInformation/RuntimeInformation.cs
+                // e.g: .NET Framework 4.7.2633.0, .NET Native, WebAssembly
+                var frameworkDescription = RuntimeInformation.FrameworkDescription;
 
-            return Parse(frameworkDescription);
+                return Parse(frameworkDescription);
```

Comment:
thanks for this. can you move the catch to only wrap the `RuntimeInformation.FrameworkDescription` so it doesnt accidentally swallow a potential bug in `Parse(frameworkDescription)`

Reply:
changed